rest_framework_simplejwt/settings.py

```diff
@@ -34,6 +34,18 @@
     'SLIDING_TOKEN_REFRESH_EXP_CLAIM': 'refresh_exp',
     'SLIDING_TOKEN_LIFETIME': timedelta(minutes=5),
     'SLIDING_TOKEN_REFRESH_LIFETIME': timedelta(days=1),
+
+    # Cookie name. Enables cookies if value is set.
+    'AUTH_COOKIE': None,
```

Comment:
I feel like this should have a better name, more verbose like `AUTH_COOKIE_NAME` like the `CSRF_COOKIE_NAME`.